examples/stm32wl/src/bin/lorawan.rs

```diff
@@ -32,11 +58,12 @@ async fn main(_spawner: embassy::executor::Spawner, p: Peripherals) {
     let ctrl3 = Output::new(p.PC5.degrade(), Level::High, Speed::High);
     let rfs = RadioSwitch::new(ctrl1, ctrl2, ctrl3);
 
-    let radio = SubGhz::new(p.SUBGHZSPI, p.PA5, p.PA7, p.PA6, NoDma, NoDma);
-
+    let radio = SubGhz::new(p.SUBGHZSPI, NoDma, NoDma);
     let irq = interrupt::take!(SUBGHZ_RADIO);
-    static mut RADIO_STATE: SubGhzState<'static> = SubGhzState::new();
-    let radio = unsafe { SubGhzRadio::new(&mut RADIO_STATE, radio, rfs, irq) };
+
+    let mut radio_config = SubGhzRadioConfig::default();
+    radio_config.calibrate_image = CalibrateImage::ISM_863_870;
+    let radio = SubGhzRadio::new(radio, rfs, irq, radio_config).unwrap();
```

Comment:
It's slightly annoying that you have to specify the region both for calibration and for the lorawan stack. I wonder if we can somehow derive this setting from region, but it's good for now I think.